versions.yaml

```diff
@@ -0,0 +1,34 @@
+# This file defines all the Istio versions supported by this operator.
+# The latest version must be listed first, so that the CRDs from the latest version
+# are copied to the config/crd directory.
+latest:
```

Comment:
Worth to add a `version` field to make it easier to change this format in the future?

Reply:
Not sure what you mean? What would that field be used for? The operator doesn't need it.

Reply:
I was just thinking about doing breaking changes in this file format in the future, and possibly having two operator versions reading the same file. Not sure it's worth the effort...

Reply:
This is an internal file that's only used by `make gen` to download charts, update the enum and docs in the Istio CRD, and collect all the istio CRDs. So, this file is only used by one version of the operator.